cfps/CFP-002-template.md

```diff
@@ -0,0 +1,84 @@
+# CFP-002: Template
+
+**SIG: SIG-NAME**
+
+**Begin Design Discussion:** YYYY-MM-DD
+
+**End Design Discussion:** YYYY-MM-DD
+
+**Cilium Release:** X.XX
+
+**Authors:** alice <alice@example.com>, bob jones <bob@example.com>
+
```

Comment:
Having some form of "State" field (like "provisional", "implementable", "implemented") is useful because it allows us to merge an early design and update it later if we want. (We can just go straight to implemented though).

This one's non-blocking though, it's just been nice in other places.

Reply:
I've created https://github.com/cilium/design-cfps/issues/4 to track this and have the conversation there